var/spack/repos/builtin/packages/py-python-lsp-jsonrpc/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPythonLspJsonrpc(PythonPackage):
+    """A Python 3.7+ server implementation of the JSON RPC 2.0 protocol."""
+
+    homepage = "https://github.com/python-lsp/python-lsp-jsonrpc"
+    pypi = "python-lsp-jsonrpc/python-lsp-jsonrpc-1.0.0.tar.gz"
+
+    maintainers = ["alecbcs"]
+
+    version("1.0.0", sha256="7bec170733db628d3506ea3a5288ff76aa33c70215ed223abdb0d95e957660bd")
+
+    depends_on("python@3.6:", type=("build", "run"))
```

Comment:
Spack only supports Python 3.7+, so you don't actually need to list this dep, up to you

Reply:
Oh I hadn't realized we only support back to 3.7 and onwards, good catch!